### PR TITLE
Build for npm publish

### PR DESCRIPTION
**Pull request by gmaclennan**
Trying to build and package for publishing npm.

Right now the packaged version does not work without babelify, gives the following error:

```
Cannot find module '../core-js/object/define-property' from '/Users/gregor/Dev/DdDev/mapfilter-new/dist'
```

We a single export from this module when it is published to npm: The React Component `<MapFilter>` that works without babel or anything else.

### CODE REVIEW

**Comment by mojodna** [comment]
Now intended to publish ES5 artifacts (`es5/*`, transpiled with babel) suitable for `require()` / `import`.

Demo: https://github.com/mojodna/mapfilter-demo

File: package.json [hunk 1]
```diff
@@ -85,15 +87,23 @@
     "start": "budo example/index.js:bundle.js --pushstate --dir example",
     "dev": "budo src/dev.js:bundle.js --pushstate --dir statics -- -p browserify-hmr",
     "test": "exit 0",
-    "build": "npm run build:statics && npm run build:js",
+    "build": "mkdirp dist && npm run -s build:js && npm run -s build:min.js",
     "build:markers": "node ./bin/build_markers.js",
     "build:statics": "copyfiles -u 1 'statics/**/*' build",
-    "build:js": "NODE_ENV=production browserify -g envify -g uglifyify src/index.js | uglifyjs -c -m > build/bundle.js"
+    "build:js": "NODE_ENV=production browserify -g envify --standalone MapFilter src/index.js > dist/mapfilter.js",
+    "build:es5": "rm -rf es5 && babel src/ -d es5/",
```
Comment (gmaclennan) [inline package.json]: let's use `rimraf`, keep windows users happy

**Comment by mojodna** [comment]
For posterity, the source of the initial symptom was that `browserify` was correctly bundling everything (including React, which would have led to further problems down the line). It stubbed out `require()`, so including `mapfilter.js` in a `<script>` tag worked. However, when used in an environment where `require()` actually existed (such as an environment built with `browserify` or just plain `node`), it would defer to that. `babel-runtime/helpers/defineProperty.js` does a relative require of `../core-js/object/define-property`, which fails to resolve appropriately in the new environment.

[`derequire`](https://github.com/calvinmetcalf/derequire) addresses the relative `require` problem, but then there were [app-specific] CSS inlining problems.

**Comment by mojodna** [comment]
@gmaclennan:

> If/when we do, for people that like script tags, https://www.bram.us/2016/08/04/distributing-your-react-component-as-a-standalone-version/ is what we need.

(`build:js` will likely get us partway there)

**Pull request merged**